src/test/java/teammates/test/cases/ui/browsertests/FeedbackMcqQuestionUiTest.java

```diff
@@ -137,9 +137,9 @@ public void testCustomizeOptions() {
 
         ______TS("MCQ: add mcq option after remove");
 
-        feedbackEditPage.clickAddMoreMcqOptionLink();
+        feedbackEditPage.clickAddMoreMcqOptionLinkForNewQuestion();
         assertTrue(feedbackEditPage.isElementPresent("mcqOptionRow-3--1"));
-        feedbackEditPage.clickAddMoreMcqOptionLink();
+        feedbackEditPage.clickAddMoreMcqOptionLinkForNewQuestion();
```

Comment:
This is part of the inconsistencies to be dealt with in #5802 -- i.e., many similar methods. Over here, I'm just renaming the method name for clarity.